notebooks/en/rag_with_hugging_face_gemma_elasticsearch.ipynb

```diff
@@ -0,0 +1,6320 @@
+{
```

Comment:
<p>Can remove "Credentials" since it is just an empty section header</p><p> </p><p>"Hugging Face"</p><p> </p><p>"...authenticate with Hugging Face to download..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>👍 TIL <code>s/Huggingface/Hugging Face</code></p>